roles/falcon_installation/tasks/preinstall.yml

```diff
@@ -22,6 +22,11 @@
 
   when: falcon_install_method == "api"
 
+- name: "CrowdStrike Falcon | Override falcon_sensor_version when falcon_sensor_update_policy_name supplied"
+  set_fact:
+    falcon_sensor_version: 0
+  when: falcon_sensor_update_policy_name != ""
+
```

Comment:
Let's move this to the sensor policy update block

Reply:
agreed